plugins/outputs/opensearch/telegrafTemplate.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
Can we rename this to `template.json`? I guess we all know that we are in telegraf land... ;-)

Reply:
done